tools/astarte_e2e/config/config.exs

```diff
@@ -23,6 +23,13 @@
 # is restricted to this project.
 use Mix.Config
 
+config :astarte_e2e, AstarteE2E.ServiceNotifier.Mailer,
+  adapter: Bamboo.ConfigAdapter,
+  chained_adapter: Bamboo.LocalAdapter,
+  hackney_opts: [
+    recv_timeout: :timer.minutes(1)
+  ]
```

Comment:
This makes sense only if you're using a mailer adapter, move it in runtime config